openapi/src/main/definitions/wacodis-schemas.yml

```diff
@@ -145,6 +145,25 @@ components:
               description: |
                 the id of the dataset within the GDI-DE catalogue
               type: string
+    DwdDataEnvelope:
+       allOf:
+        - $ref: '#/components/schemas/AbstractDataEnvelope'
+        - type: object
+          required:
+          - layerName
+          - parameters
+          properties:
+            layerName:
+              description: |
+                the name of the queried layer
+              type: string
+            parameters:
+              description: |
+                the output attributes
+              type: list
+                enum: ['GEOM', 'BEOBACHTER_GUID', 'STATION_ID', 'STATION_NAME', 'ZEITSTEMPEL', 'WERT', 'QUALITAET_NIVEAU', 'QUALITAET_BYTE']
+                items:
+                type: string
```

Comment:
"Parameter" should rather be the climate parameter (e.g. "temperature","wind") that will be provided by the DWD service corresponding to the "layerName" attribute.